replications/internal/queue_management.go

```diff
@@ -170,7 +169,7 @@ func (rq *replicationQueue) SendWrite(dp func([]byte) error) bool {
 		// An error here indicates an unhandlable error. Data is not corrupt, and
 		// the remote write is not retryable. A potential example of an error here
 		// is an authentication error with the remote host.
-		if err = dp(scan.Bytes()); err != nil {
+		if err = rq.remoteWriter.Write(context.Background(), scan.Bytes()); err != nil {
```

Comment:
This context isn't doing much right now, but I think may need to be used as part of a follow-up issue: https://github.com/influxdata/influxdb/issues/22944